Samples/Forms/LibVLCSharp.Forms.Sample/MainPage.xaml.cs

```diff
@@ -1,44 +1,30 @@
-﻿using LibVLCSharp.Shared;
-using System.Diagnostics;
+﻿using System.Diagnostics;
 using Xamarin.Forms;
 
 namespace LibVLCSharp.Forms.Sample
 {
     public partial class MainPage : ContentPage
     {
-        MainViewModel _vm;
-
         public MainPage()
         {
             InitializeComponent();
-        }
-
-        protected override void OnAppearing()
-        {
-            base.OnAppearing();
-
             videoView.MediaPlayerChanged += VideoView_MediaPlayerChanged;
-            videoView.Loaded += VideoView_Loaded;
-
-            _vm = BindingContext as MainViewModel;
-            _vm.PropertyChanged += Vm_PropertyChanged;
-            _vm.Initialize();
+            BindingContext.PropertyChanged += BindingContext_PropertyChanged;
+            BindingContext.Play();
         }
 
-        private void VideoView_Loaded(object sender, System.EventArgs e)
-        {
-            _vm.MediaPlayer.Play();
-        }
+        private new MainViewModel BindingContext => (MainViewModel)base.BindingContext;
```

Comment:
Not a huge fan of `new` overrides. could we call that another way? (I tend to call that typed property `ViewModel`)

Reply:
OK, I'm going to rename it to ViewModel.